### PR TITLE
[Non_Sanity_Testing] Added scripts to update config for non sanity test cases

### PR DESCRIPTION
**Pull request by VinashakAnkitAman**
## Title
[Non_Sanity_Testing] Added scripts to update config for non sanity test cases

## Summary
When adding all non-sanity test cases as part of defs.mk file under PR #5840, the IPv6 test cases and AGW offload test cases were failing, because they needed some services to be enabled. This test case adds a generic script to do all configuration modifications together for all non-sanity test cases.

## Test Plan
Verified by wrapping the non-sanity test cases under these scripts.
 

Signed-off-by: VinashakAnkitAman <ankit.aman@radisys.com>

### CODE REVIEW

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py [hunk 1]
```diff
@@ -627,6 +627,7 @@ class MagmadUtil(object):
     config_update_cmds = Enum("config_update_cmds", "MODIFY RESTORE")
     apn_correction_cmds = Enum("apn_correction_cmds", "DISABLE ENABLE")
     health_service_cmds = Enum("health_service_cmds", "DISABLE ENABLE")
+    enable_disable_options = Enum("enable_disable_options", "DISABLE ENABLE")
```
Comment (ardzoht) [inline lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: I would prefer having separate Enums for both HA and IPv6 configurations, let's separate this and call each one `ipv6_config_cmds` and `ha_service_cmds`

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py [hunk 2]
```diff
@@ -874,6 +873,105 @@ def config_health_service(self, cmd: health_service_cmds):
...
+            "grep 'ipv6_solicitation' /etc/magma/pipelined.yml | wc -l"
+        )
+        retVal = self.exec_command_output(ipv6_config_status_cmd).rstrip()
+
+        if cmd.name == MagmadUtil.enable_disable_options.ENABLE.name:
+            if retVal != "0":
+                print("IPv6 solicitation service is already enabled")
+                return 0
+            else:
+                ipv6_config_cmd = "sed -i \\\"/startup_flows/a \ \ 'ipv6_solicitation',\\\" /etc/magma/pipelined.yml"
```
Comment (ardzoht) [inline lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]:
```suggestion
                ipv6_update_config_cmd = "sed -i \\\"/startup_flows/a \ \ 'ipv6_solicitation',\\\" /etc/magma/pipelined.yml"
```

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py [hunk 3]
```diff
@@ -874,6 +873,105 @@ def config_health_service(self, cmd: health_service_cmds):
...
+        else:
+            if retVal == "0":
+                print("IPv6 solicitation service is already disabled")
+                return 0
+            else:
+                ipv6_config_cmd = (
```
Comment (ardzoht) [inline lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]:
```suggestion
                ipv6_update_config_cmd = (
```

**Review by ardzoht — CHANGES_REQUESTED** [review]
minor changes on Enum for cmd options and convention naming, thanks for this!

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py (hunk 2 above)
Comment (VinashakAnkitAman) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: Done

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py (hunk 3 above)
Comment (VinashakAnkitAman) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: Done

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py (hunk 1 above)
Comment (VinashakAnkitAman) [reply lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py]: Done

**Review by ardzoht — APPROVED** [review]

**Review by ulaskozat — APPROVED** [review]
LGTM

**Pull request merged**